src/sentry/api/endpoints/project_codeowners_details.py

```diff
@@ -0,0 +1,66 @@
+import logging
+from django.http import Http404
+
+from rest_framework import status
+from rest_framework.response import Response
+
+from sentry.api.bases.project import ProjectEndpoint
+from sentry.api.serializers import serialize
+from sentry.models import ProjectCodeOwners
+
+from .project_codeowners import ProjectCodeOwnerSerializer
+
+from sentry.api.endpoints.project_ownership import ProjectOwnershipMixin
+
+logger = logging.getLogger(__name__)
+
+
+class ProjectCodeOwnersDetailsEndpoint(ProjectEndpoint, ProjectOwnershipMixin):
+    def convert_args(
+        self, request, organization_slug, project_slug, codeowners_id, *args, **kwargs
+    ):
+        args, kwargs = super().convert_args(
+            request, organization_slug, project_slug, *args, **kwargs
+        )
+        try:
+            kwargs["codeowners"] = ProjectCodeOwners.objects.get(
+                id=codeowners_id,
```

Comment:
Should this query be scoped to the organization or projects that the current user is in? Right now I could delete mappings in another organization by enumerating the id values in the URL, but providing my project & organization.

Reply:
@markstory ohhh I didn't think of that! I'll update the query for this.